windows/visual-studio/packer.yaml

```diff
@@ -43,6 +50,51 @@ builders:
     run_volume_tags:
       circle_build_url: '{{ user "circle_build_url" }}'
 
+  - type: googlecompute
```

Comment:
You will need a `subnetwork` field for GCP networks that don't use auto-subnetting which will be true for many if not most users users